### PR TITLE
[COD-3] Add gem devise and config model user

### PR DESCRIPTION
**Pull request by SilaEwGenii**
https://legolas.atlassian.net/browse/COD-3
 Add devise and add create model User.

### CODE REVIEW

**Review by LorDDeadWolf — CHANGES_REQUESTED** [review]
Just set all checkboxes

**Review by LorDDeadWolf — APPROVED** [review]

File: test/models/user_test.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class UserTest < ActiveSupport::TestCase
```
Comment (npupko) [inline test/models/user_test.rb:3]: We don't need test files, because we're using RSpec now

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
```
Comment (npupko) [inline app/models/user.rb:1]: Please, add basic RSpec test to check that we able to create a user

**Review by npupko — CHANGES_REQUESTED** [review]

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+
+  it "create users" do
+    user1 = User.create({name:'Pasha', email:'test@mail.ru', password: 'testtest'})
```
Comment (npupko) [inline spec/models/user_spec.rb:6]: It's better not to create local variables inside specs and instead use `let` and `subject` directives:
- https://www.betterspecs.org/#let
- https://www.betterspecs.org/#subject

**Review by npupko — CHANGES_REQUESTED** [review]

**Review by npupko — APPROVED** [review]
LGTM, great job!

**Review by LorDDeadWolf — COMMENTED** [review]
Krasava!

**Pull request merged**